superdesk/publish/transmitters/ftp.py

```diff
@@ -51,13 +54,67 @@ def _transmit(self, queue_item, subscriber):
 
         try:
             with ftp_connect(config) as ftp:
+
+                if config.get('push_associated', False):
```

Comment:
is this config only available with ninjs formatter?

Reply:
This is a config setting for the FTP transmitter, set in the config for the transmitter in client-core.
There is an assumption that the formatted item will be ninjs, but if there are no associations in whatever format it should not be a problem!

Reply:
I'm not sure about this `item = json.loads(queue_item['formatted_item'])` in case of eg newsml, maybe it should have some `try/catch` on it

Reply:
Good point! I've put a try catch on it.